sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -26,11 +26,69 @@ import { WebSocketImpl } from 'rhea-promise';
 
 export { AccessToken }
 
+// @public
+export interface AmqpAnnotatedMessage {
+    applicationProperties?: {
+        [key: string]: any;
+    };
+    body: any;
+    deliveryAnnotations?: {
+        [key: string]: any;
+    };
+    footer?: {
+        [key: string]: any;
+    };
+    header?: AmqpMessageHeader;
+    messageAnnotations?: {
+        [key: string]: any;
+    };
+    properties?: AmqpMessageProperties;
+}
+
+// @public
+export const AmqpAnnotatedMessage: {
+    fromRheaAmqpMessage(msg: AmqpMessage): AmqpAnnotatedMessage;
+};
+
 export { AmqpMessage }
 
-export { AmqpMessageHeader }
+// @public
+export interface AmqpMessageHeader {
```

Comment:
`AmqpMessageHeader` was previously reflecting the snake casing and was from rhea-promise. Now it reflects camel casing and is defined in core-amqp. This will require us to update the major version. Let's do that in this PR. 2.0.0-beta.1